components/src/preact/MutationAnnotationsContext.tsx

```diff
@@ -35,13 +49,19 @@ export const MutationAnnotationsContextProvider: FunctionalComponent<
             new Set(annotation.aminoAcidMutations).forEach((code) => {
                 addAnnotationToMap(aminoAcidMap, code, annotation);
             });
+            new Set(annotation.nucleotidePositions).forEach((position) => {
+                addAnnotationToMap(nucleotidePositions, `${position}`, annotation);
+            });
+            new Set(annotation.aminoAcidPositions).forEach((position) => {
+                addAnnotationToMap(aminoAcidPositions, `${position}`, annotation);
```

Comment:
Those are already strings:
```suggestion
                addAnnotationToMap(nucleotidePositions, position, annotation);
            });
            new Set(annotation.aminoAcidPositions).forEach((position) => {
                addAnnotationToMap(aminoAcidPositions, position, annotation);
```